bedrock/base/templates/base-new.html

```diff
@@ -0,0 +1,149 @@
+{% from "macros.html" import share_cta with context %}
```

Comment:
Not a blocker for this PR, but with this change we will now have three base templates. There seems to be a lot of duplication here (particularly all the stuff in the `<head>`). Is it worth filing a follow up bug to consolidate / share some of this between templates?